config/targets.conf

```diff
@@ -882,3 +882,17 @@ z28pro				legacy			buster		cli			stable	yes
 z28pro				legacy			focal		desktop			stable	yes
 z28pro				legacy			focal		cli			stable	yes
 z28pro				legacy			focal		desktop			stable	yes
+
+# Firefly Station M1
+
+firefly-m1			current			bullseye	cli			stable	yes
```

Comment:
I would stick the downloadable images to buster for now.

Reply:
> I would stick the downloadable images to buster for now.

Buster, in comparison with Bullseye, has old packages, which negatively affects the use of new functions (mesa, browser etc). I regularly compare their work and  my opinion Bullseye works better.

Reply:
Ok, this is CSC anyway. Let's keep it with bullseye.